src/editor/editor-contract-test.ts

```diff
@@ -406,4 +407,26 @@ console.log("hello");
 
     assert.sameDeepMembers(result, files);
   });
+
+  test("should return the list of code references in same file", async () => {
+    const code = `function add(num1, num2) {
+return num1 + num2;
+}
+add(1, 2);
+`;
+    const editor = await createEditorOn(code);
+    const file =
+      __dirname + "/adapters/vscode-editor-tests/abracadabra-vscode-tests.ts";
```

Comment:
🍰  Is it necessary to use such a specific file? 
Could we make it relative, like `const filePath = new RelativePath('./code-references.playground.ts')`?

Reply:
I just saw your changes, the main issue was with TestRunner itself 👍🏽